docs/TEAM.md

```diff
@@ -84,3 +82,5 @@
 |                                                                 | Kingsley                                      | 3+ yrs of experience. Based in Nigeria and here to help :man_technologist:                         |
 |                                                                 | [Pratham](https://github.com/prasoonpratham)  | 2+ experience in JavaScript, always open to help :smile:                                           |
 |                                                                 | [Sarath](http://http://www.sarathnagaraj.ca/) | 6+ experience in JavaScript, Currently working on a side project using ReactJS :man_technologist:  |
+
+|<img src="" width="80px" height="80px"> | [Sachin](https://github.com/toocoolsachin) | I'm from India, experience in JavaScript, Node, React, Html, Css.
```

Comment:
@toocoolsachin add "|" at the last and preview your changes.

![image](https://user-images.githubusercontent.com/72256599/97422144-6ac11f80-1933-11eb-8847-a68916f77e1d.png)